### ISSUE TITLE
Select All on focus change

### ISSUE THREAD

**Issue by justdan0227** [neoneye/SwiftyFORM#64]
Is there a way to set a field as SELECT ALL when focus change to it?

**Comment by neoneye** [comment]
What do you mean with "SELECT ALL", like a text where all text is selected?

**Comment by justdan0227** [comment]
Yes sorry so I would like to have each field to be SELECT ALL when focus goes to it so you can just click and enter vs having to press the X to clear the current value.

**Comment by neoneye** [comment]
In `TextFieldFormItem` and `TextViewFormItem`, maybe add an enum ala `tapBehavior = .selectText`. 

In `TextFieldCell.handleTap()` deal with the `tapBehavior` and select all the text.

PR is welcome.
